rackcity/views/rackcity_utils.py

```diff
@@ -1,5 +1,37 @@
 from rackcity.models import ITInstance, ITModel, Rack
 from rackcity.api.objects import RackRangeSerializer
+from rackcity.api.serializers import RecursiveITInstanceSerializer, RackSerializer
+from http import HTTPStatus
+from django.http import JsonResponse
+
+
+def get_rack_detailed_response(racks):
+    if racks.count() == 0:
+        return JsonResponse(
+            {"failure_message": "There are no existing racks within this range. "},
+            status=HTTPStatus.BAD_REQUEST,
+        )
```

Comment:
we can refactor this later when we dig into user-friendliness of error messages, but I don't think a user's attempt to view all should show the same "range" failure message if there are no racks

Reply:
sounds good - tracking this for a future refactor